workspaces/src/rpc/client.rs

```diff
@@ -366,9 +378,11 @@ impl Client {
         result
     }
 
-    pub(crate) async fn wait_for_rpc(&self) -> anyhow::Result<()> {
+    pub(crate) async fn wait_for_rpc(&self) -> Result<(), RpcError> {
         let timeout_secs = match std::env::var("NEAR_RPC_TIMEOUT_SECS") {
-            Ok(secs) => secs.parse::<usize>()?,
+            // hard fail on not being able to parse the env var, since this isn't something
+            // the user should handle with the library.
+            Ok(secs) => secs.parse::<usize>().unwrap(),
```

Comment:
hmm, normally for user input errors, wouldn't we want to hardfail though? Since that gives immediate feedback, and I just see this being a case that won't get handled at all. It just feels weird when we go to handle it, since it would just be used for potentially cleanup. But then again, for things like dev console, we'd want to package errors as cleanly as possible. I do see value with having it in that case. What would be a good name for the class of errors of these? `InputError`? Or should we put it in something like `ParseError`?

Reply:
The issue is that this might be used other than just with a test, and this wait_for_rpc would panic when someone might want to gracefully handle errors and re-start the sandbox or something else. I don't see how this would want to be handled differently than the error below.

At least, Daniyar's suggestion of adding an error message with this feels necessary.